File: JS/decrypt.js
/********************************

Use: "😝😅😒😒😄😍😅".decrypt()
=> 'MESSAGE'
********************************/
String.prototype.decrypt = function () {
	const emojis ='😄,😃,😀,😊,😅,😉,😍,😘,😚,😗,😙,😜,😝,😛,😳,😁,😔,😌,😒,😞,😣,😢,😂,😭,😎,😈'.split(",")

	const alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ".split("")
	let out = ""
	for (let o of this) {
		out += alphabet[emojis.indexOf(o)]
	}
	return out;
};
module.exports = text => text.decrypt();

File: py/decrypt.py
mydict = {"A":"😄","B":"😃","C":"😀","D":"😊","E":"😅","F":"😉","G":"😍","H":"😘","I":"😚","J":"😗","K": "😙","L":"😜","M":"😝","N":"😛","O":"😳","P":"😁","Q":"😔","R":"😌","S":"😒","T":"😞","U": "😣","V":"😢","W":"😂","X":"😭","Y":"😎","Z":"😈"}

def decrypt(message):
  """ Use : decrypt("😝😅😒😒😄😍😅")
=> 'MESSAGE'
  """
  output = ""
  for character in message:
    output += list(mydict.keys())[list(mydict.values()).index(character)]
  return output
